tex/main.tex

```diff
@@ -14,10 +14,24 @@
 \section{Introduction}
 
 \section{Literature Review}
+Evolutionary game theory has long been at the forefront of the study of emergent cooperation, showing how a strategy that is seemingly unfavourable for the individual, but beneficial to the collective, can become dominant in systems of rational actors. This is often shown through the study of games such as the iterated prisoner's dilemma\cite{IPD1,IPD2} and the snowdrift game\cite{Snowdrift1}. These examples, however, only allow for pairwise interactions, which does not accurately represent many real-world scenarios. Therefore, we look at a common game used to model the sharing of resources and interactions between multiple people - the public goods game.\\
```

Comment:
So I'm about to make a super simple suggestion which is just to add a blank line. But because of the huge single line paragraph it will be a complicated diff.

You can set this in VS code so that it will automatically restrict line lengths to a number (80 is common, arguably 120 is fine: short is good -- it makes diffs clearer).